src/lcd.cpp

```diff
@@ -103,10 +103,18 @@ void LCD_ButtonEnable(lcd_t& lcd, uint8_t enable)
 
 void LCD_SetContrast(lcd_t& lcd, uint8_t contrast)
 {
-    if (contrast > 16)
-        contrast = 16;
-    else if(contrast < 1)
-        contrast = 1;
+    if (lcd.mcu.is_jv880)
```

Comment:
`lcd` has a member which is a pointer to a `mcu_t` object
So the use is 
`lcd.mcu->is_jv880`